docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,405 @@
+#### Set Up the Environment
+
+1. Set the project, replacing the `{PROJECT_ID}` placeholder with your own project ID (see [Confidential Space Account Setup](#confidential-space-account-setup)):
+
+   ```
+   gcloud config set project {PROJECT_ID}
+   ```
+
+2. Configure the environment for Terraform:
+
+   ```
+   gcloud auth application-default login
+   ```
+
+#### Run Terraform
+
+Run the following:
+
+```
+terraform init
+terraform apply
+```
+
+#### Test
+
+1. Get the public IP address for the load balancer:
+
+   ```
+   terraform output load-balancer-ip
+   ```
+
+2. Check health at the health check endpoint (for details, see [Test Using the Health Check Endpoint](#test-using-the-health-check-endpoint)):
+
+   ```
+   http://{PUBLIC_IP}/ops/healthcheck
+   ```
+
+#### Clean Up
+
+Remove all resources created by Terraform:
+
+```
+terraform destroy
+```
+
+#### Inputs
+
+(**GWH_YS18 I need more data re the inputs and outputs so we can have some little intro line to these tables. Is this something they clean up? Or something that's required? Please help with some explanation for both inputs and outputs.**)
+
+| Name               | Description | Type     | Default             | Required |
+|--------------------|-------------|----------|---------------------|:--------:|
+| project_id         | n/a         | `string` | n/a                 |   yes    |
+| service_account    | n/a         | `string` | n/a                 |   yes    |
+| uid_operator_image | n/a         | `string` | n/a                 |   yes    |
+| uid_api_token      | n/a         | `string` | n/a                 |   yes    |
+| region             | n/a         | `string` | `"asia-southeast1"` |    no    |
+| network_name       | n/a         | `string` | `"uid-operator"`    |    no    |
+| uid_machine_type   | n/a         | `string` | `"n2d-standard-16"` |    no    |
+| uid_deployment_env | n/a         | `string` | `"integ"`           |    no    |
+| max_replicas       | n/a         | `number` | `5`                 |    no    |
+| min_replicas       | n/a         | `number` | `1`                 |    no    |
+| debug_mode         | n/a         | `bool`   | `false`             |    no    |
+
+#### Outputs
+
+| Name             | Description |
+|------------------|-------------|
+| load-balancer-ip | n/a         |
+
+#### Terraform Template&#8212;Notes
+
+1. You might want to change the load balancer from HTTP to HTTPS. To do this, follow these steps: (**GWH_YS19 surely they would have to use HTTPS? Should this be higher profile than a note at the end?**)
+
+   - Provide your cert via terraform following this page:
+  [google_compute_ssl_certificate](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_ssl_certificate.html)
+
+   - Then add the following configuration values in `module "gce-lb-http"` (**GWH_YS20 is the syntax correct here? Is it a name/value pair? Don't understand this. Module not previously mentioned. It's a module of what?**)
```

Comment:
In the beginning of this section, we should also provide a link to the template files ( in my PR):
https://github.com/IABTechLab/uid2-operator/pull/212/files#diff-154a1adda6944684fff3b5e1fc4a5afa2ae6173790c206ae062f4b173f585d05

Customer should download the 
 - `main.tf` (template) 
 - `variables.tf` (input definition) 
 - `outputs.tf` (output definition)
files, before they run `terraform apply` etc.